src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsresettlementpassportapi/data/cvlapi/LicenceSummaryDTO.kt

```diff
@@ -2,29 +2,8 @@ package uk.gov.justice.digital.hmpps.hmppsresettlementpassportapi.data.cvlapi
 
 data class LicenceSummary(
   val licenceId: Long,
-  val licenceType: String,
   val licenceStatus: String,
-  val nomsId: String,
```

Comment:
I assume we arnt using any of these field anywhere?

Reply:
That's right - only the id, status and createdDate are being used.